src/main/java/chris/draft/random/RandomDraft.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package chris.draft.random;
+import sx.blah.discord.handle.obj.IMessage;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+/**
+ *
+ * @author JCH
+ */
+
+public class RandomDraft {
+    private final String pathToCivFile = "src/main/resources/civilizations.txt";
+    private ArrayList<String> civs;
+    private final int DEFAULT_SIZE = 12;
+    
+    RandomDraft()
+    {
+        Load(pathToCivFile);
+    }
+    
+    RandomDraft(String pathToFile)
+    {
+        Load(pathToFile);
+    }
+    
+    public void Load(String pathToFile)
+    {
+        try
+        {
+            File file = new File(pathToFile);
+            FileReader fileReader = new FileReader(file);
+            BufferedReader bufferedReader = new BufferedReader(fileReader);
```

Comment:
Should use nio:

```
try (BufferedReader bufferedReader = Files.newBufferedReader(path)) {
  // Do read operation here
}
```

So that it will auto close the reader if there is an error.